nomad-core/src/aws.rs

```diff
@@ -0,0 +1,60 @@
+use once_cell::sync::OnceCell;
+use rusoto_core::{
+    credential::{AutoRefreshingProvider, ProvideAwsCredentials},
+    Client,
+};
+use rusoto_kms::KmsClient;
+
+static CLIENT: OnceCell<Client> = OnceCell::new();
+static KMS_CLIENT: OnceCell<KmsClient> = OnceCell::new();
```

Comment:
Is there a reason not to use `lazy_static!` here?

Reply:
biggest is that this is all async

next is that we should prefer non-macro once_cell to macro lazy_static in all cases except `no_std` :)

Reply:
Gotcha, makes sense